examples/example_plugin/example_plugin/graphql/types.py

```diff
@@ -1,6 +1,6 @@
 import graphene_django_optimizer as gql_optimizer
 
-from dummy_plugin.models import AnotherDummyModel
+from example_plugin.models import AnotherDummyModel
 
 
 class AnotherDummyType(gql_optimizer.OptimizedDjangoObjectType):
```

Comment:
Please rename this to `AnotherExampleModelType` (not `AnotherExampleType`) as well.